blenderproc/python/writer/CocoWriterUtility.py

```diff
@@ -225,14 +236,18 @@ def generate_coco_annotations(inst_segmaps, inst_attribute_maps, image_paths, su
             instances = np.delete(instances, np.where(instances == 0))
             for inst in instances:
                 if inst in instance_2_category_map:
+                    inst_attributes = next((item for item in inst_attribute_map if item["idx"] == inst), None)
```

Comment:
Can we maybe exclude `idx` and `name` here? They should be part of the coco annotations already anyway right?